### DISCUSSION TITLE
How to check current avaliable product quantity ?

### DISCUSSION THREAD

**Discussion by vikingcodes** [amzn/selling-partner-api-models#4049] [General] [answered]
We need to check current avaliable product quantity, Is any report contains product quantity data ?

**Comment by arpetris** [comment]
We use the GET_MERCHANT_LISTINGS_ALL_DATA report. You can find all the inventory reports here https://developer-docs.amazon.com/sp-api/docs/report-type-values-inventory. 

There are also reports for just FBA Inventory.

**Comment by vikingcodes** [comment, marked as answer]
Thanks @arpetris I found "pending-quantity" , I was using "GET_MERCHANT_LISTINGS_ALL_DATA" but missed that attribute.
